.eslintrc

```diff
@@ -18,6 +18,13 @@
     // However, it will not warn you if you access `this` in a static method
     // accidentally (which would most likely be a mistake).
     "class-methods-use-this": "off",
+    "comma-dangle": ['error', {
+      arrays: 'always-multiline',
+      objects: 'always-multiline',
+      imports: 'always-multiline',
+      exports: 'always-multiline',
+      functions: 'ignore',
```

Comment:
This one we might also want to revisit. Since it if we follow the default rules you get a warning if args are spread out on multiple lines without a trailing comma. Support for that in funcs is in es2017 apparently.